lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -812,10 +812,8 @@ def default_ami_for_region(region, criteria = {})
 
     def create_winrm_transport(machine_spec, machine_options, instance)
       remote_host = determine_remote_host(machine_spec, instance)
+      winrm_transport_options = machine_options[:winrm_transport]
```

Comment:
That was brought over from https://github.com/chef/chef-provisioning-azure/blob/master/README.md#windows we probably need to decide on a cross provider way to specify all the azure stuff.... I don't think it goes into bootstrap, due to the fact that the way you connect might change later (after the first convergence, you might add ssh for example).